### ISSUE TITLE
Assertion failed: scales_input.is_weights()

### ISSUE THREAD

**Issue by ghylander** [onnx/onnx-tensorrt#708]
## Description

The full error meesage is the following:

ERROR: builtin_op_importers.cpp:3460 In function importUpsample: 
[8] Assertion failed: scales_input.is_weights()

the error comes out after a few layers have been translated

this fails with a model with the following (deviation from working exports):
- Input shape is (None, None, 3)
- there is a tf.keras.layers.experimental.preprocessing.Resizing() layer
- there is a tf.keras.layers.experimental.preprocessing.CenterCrop() layer

## Environment

**TensorRT Version**: 7.1.3
**ONNX-TensorRT Version / Branch**: 7.1.3-1
**GPU Type**: jetson nano
**Nvidia Driver Version**: n/a
**CUDA Version**: 10.2
**CUDNN Version**: 8.0
**Operating System + Version**:  Ubuntu 18.04
**Python Version (if applicable)**: 2.8.5
**TensorFlow + TF2ONNX Version (if applicable)**:  tf 2.5, 1.8.5

## Steps To Reproduce

/usr/src/tensorrt/bin/trtexec --onnx={onnxFilePath} --fp16 --workspace=1024 --warmUp=2 --dumpProfile --verbose --explicitBatch --saveEngine={engineFilePath}

**Comment by ghylander** [comment]
forgot to mention:
yes, i tried folding the onnx model
the folding operation itself was succesful, but when running exporting to trt it gives the same error

**Comment by kevinch-nv** [comment]
Can you attach the ONNX model?
